core/src/modules/ops/profile.ts

```diff
@@ -593,10 +593,15 @@ export namespace ProfileOps {
   /**
    * The method you'll be using to create profiles with arbitrary data.
    * Hash looks like {email: "person@example.com", id: 123}
+   *
+   * This method today always returns a profile by finding it or making a a new one... is that right?.
```

Comment:
I believe that's right! Though now it can also throw an error if the schedule's source doesn't own the unique property being set